### ISSUE TITLE
Invalid sdk value for 8.1.0-alpha.7

### ISSUE THREAD

**Issue by luke-hawk** [NativeScript/ns-v8ios-runtime#128]
I've just tested the latest version `8.1.0-alpha.7` and would like to report that my archive (xcode 12.4) has been declined for the following reason:

```
ITMS-90512: Invalid sdk value - The value provided for the sdk portion of LC_VERSION_MIN_IPHONEOS in dermanosticmobile.app/Frameworks/NativeScript.framework/NativeScript is 15.0 which is greater than the maximum allowed value of 14.7.

ITMS-90512: Invalid sdk value - The value provided for the sdk portion of LC_VERSION_MIN_IPHONEOS in dermanosticmobile.app/Frameworks/TKLiveSync.framework/TKLiveSync is 15.0 which is greater than the maximum allowed value of 14.7.
```

Seems like a xcode build issue with `8.1.0-alpha.7`?

**Comment by NathanWalker** [comment]
Thanks for mentioning this @luke-hawk - I believe the alpha was built with Xcode beta so we'll make sure the final 8.1 release shortly is built with the stable release which should resolve that. We'll keep you posted here. For now you can probably use the released version until we have 8.1 out.

**Comment by NathanWalker** [comment]
@luke-hawk we published 8.1 now if you could switch your package to @nativescript/ios '8.1.0', clean and upload to store - let us know if further issue.

**Comment by luke-hawk** [comment]
Thanks for your response and for releasing 8.1 @NathanWalker🎉 – the problem seem to be resolved. I uploaded a new build to the store and haven't gotten the above error. Cheers!
